### ISSUE TITLE
Suprising surface reconstruction

### ISSUE THREAD

**Issue by pyp22** [simonfuhrmann/mve#213]
Hi Simon, i met today a strange behavior . Here is a snapshot of the L2 surface. It's beeing there since the pset.

![snapshot00](https://cloud.githubusercontent.com/assets/8407357/13899078/1c4823c2-ede5-11e5-9459-8c6dd0680736.png)

Best regards,

PY

**Comment by simonfuhrmann** [comment]
Hi Pierre,

that looks indeed strange, like as the camera parameters are not properly applied. Can you take a look at the scene in UMVE, and see if the cameras are in reasonable positions around the object (I bet they are, otherwise MVS would not have worked). Also, can you triangulate a couple of depth maps and see if they line up with the SfM point cloud?

Also, did you run SfM after MVS, i.e., changed the camera parameters after MVS? This would explain this output.

Simon

**Comment by pyp22** [comment]
All cams positions seem to be ok. I ran the "classic" way of using the pipeline wihtout changing or modify anything. I reran the whole dataset from scratch to check, including openMVG. I was unable to reproduce the result. Computer might have had a fit of hiccups. I was simultaneously running for another reconstruction while running mve. Some garbage in memory while working on both jobs ? I'll retry that on next ones to see. Closing this by now. Thanks Simon.
